src/Main.java

```diff
@@ -32,4 +35,28 @@ public void setHeight(double height) {
             this.height = height;
         }
     }
+
+    public static class Square extends Rectangle {
+        protected double side;
+
+        public Square(double side) {
+            super(side, side);
+            this.side = side;
+        }
+        public double getSide() {
+            return side;
+        }
+
+        public void setSide(double side) {
```

Comment:
You should use @override because we have this in father class; in this way, you are not following SOLID Rules.

Reply:
setSide is not is Rectangle